v2/client.go

```diff
@@ -79,6 +79,8 @@ func NewClient(config *ClientConfiguration) (Client, error) {
 		c.AuthConfig = config.AuthConfig
 	}
 
+	c.Verbose = config.Verbose
```

Comment:
Fine by me, though nit: could you move this up into the declaration of `c` at L62? No need for it to be it's own isolated section.

Reply:
Yep